docs/docs/low_level_app.md

```diff
@@ -0,0 +1,159 @@
+This contract will live in the blockchain and manage the different versions of our smart contracts.
+
+```js
+const { App } = require('zos-lib')
+const initialVersion = '0.0.1'
+await App.deploy(initialVersion)
+```
+
+### 3. Create the contract
+
+An initial version of the Solidity contract could look like:
+
+```sol
+pragma solidity ^0.4.21;
+
+import "openzeppelin-zos/contracts/ownership/Ownable.sol";
+import "openzeppelin-zos/contracts/math/SafeMath.sol";
+
+contract DonationsV0 is Ownable {
+  using SafeMath for uint256;
+
+  // Keeps a mapping of total donor balances
+  mapping(address => uint256) public donorBalances;
+
+  function donate() payable public {
+    require(msg.value > 0);
+
+    // Update user donation balance
+    donorBalances[msg.sender] = donorBalances[msg.sender].add(msg.value);
+  }
+
+  function getDonationBalance(address _donor) public view returns (uint256) {
+    return donorBalances[_donor];
+  }
+
+  function withdraw(address _wallet) onlyOwner {
+    // Withdraw all donated funds
+    _wallet.transfer(this.balance);
+  }
+}
+```
+
+Now, let's deploy the first version of our contract. To do so, we register its implementation in the `App` and then request to create a new proxy for it:
+
+```js
+const contractName = "Donations";
+const DonationsV0 = Contracts.getFromLocal('DonationsV0')
+await app.setImplementation(DonationsV0, contractName)
+const donationsV0 = await app.createProxy(DonationsV0, contractName, 'initialize', [owner])
+```
+
+Remember that the proxy is the contract that will receive the calls and hold the storage, while delegating its behavior to the implementation contract, enabling us to upgrade it.
+
+### 4. Link a standard library
+
+Now let's suppose we want to give some sort of retribution to the donors, so we mint new [ERC721](http://erc721.org/) cryptocollectibles for each donation. 
+
+For this, we link the [ZeppelinOS standard library](https://www.npmjs.com/package/openzeppelin-zos) to our application by running:
+
+```sh
+npm install openzeppelin-zos --save
+```
+
+Done! Now we can easily mint non-fungible tokens from our smart contract:
+
+```sol
+pragma solidity ^0.4.21;
+
+import "./DonationsV0.sol";
+import "openzeppelin-zos/contracts/token/ERC721/MintableERC721Token.sol";
+
+contract DonationsV1 is DonationsV0 {
+  using SafeMath for uint256;
+
+  MintableERC721Token public token;
+  uint256 public numEmittedTokens;
+
+  function setToken(MintableERC721Token _token) external {
+    require(_token != address(0));
+    require(token == address(0));
+    token = _token;
+  }
+
+  function donate() payable public {
+    super.donate();
+    token.mint(msg.sender, numEmittedTokens);
+    numEmittedTokens = numEmittedTokens.add(1);
+  }
+}
+```
+
+Notice that by doing this, our contract will interact directly with the on-chain ZeppelinOS standard library, so there is no need to deploy nor maintain the `MintableERC721Token` contract ourselves, as we rely on the ZeppelinOS network.
```

Comment:
~, as we rely on the ZeppelinOS network~

Reply:
Why? My rationale was to make explicit they're not relying on us as a company, by running our privative code and to introduce the concept of the network in the way.

Reply:
We nowhere talk about a "ZeppelinOS network", and I don't see why to introduce here, or what it really means. I believe this last part as it is doesn't add to the phrase, and doesn't convey the meaning you mention in your comment. If you want to say that I'd add a separate, more explicit phrase.